### PR TITLE
Reach Comparison Chart

### PR DESCRIPTION
**Pull request by tiggreen**
### Purpose

Reach Comparison Chart that shares a lot of things with Audience Comparison! 

#### Staging Deployment

_This repo is CI/CD enabled and staging deployment should be available at:
https://<branch_name>.analyze.buffer.com :smile:_

### CODE REVIEW

File: packages/audience-comparison-chart/components/AudienceComparisonChart/story.jsx [hunk 1]
```diff
@@ -11,6 +11,7 @@ const profileTotals = [
     currentPeriodDiff: 20,
     profileId: '1234abcd',
     username: 'Buffer',
+    servuce: 'facebook',
```
Comment (msanroman) [inline packages/audience-comparison-chart/components/AudienceComparisonChart/story.jsx]: Woop! This should be `service` :D

File: packages/audience-comparison-chart/components/AudienceComparisonChart/index.jsx [hunk 2]
```diff
@@ -1,66 +1,20 @@
...
-      </ContentContainer>
-      {footer}
-    </div>
+}) =>
+  (
+    <ComparisonChartWrapper
+      profilesMetricData={profilesMetricData}
+      profileTotals={profileTotals}
+      loading={loading}
+      chartName={'Audience'}
```
Comment (msanroman) [inline packages/audience-comparison-chart/components/AudienceComparisonChart/index.jsx]: Wondering if we could make `Audience` a constant?

File: packages/audience-comparison-chart/middleware.js [hunk 3]
```diff
@@ -8,24 +8,26 @@ export default store => next => (action) => { // eslint-disable-line no-unused-v
     case profileActionTypes.COMPARE_PROFILES: {
       const selectedProfileIds = action.selectedProfiles.map(p => p.id);
       dispatch(actions.fetch({
-        name: 'audience_comparison',
+        name: 'comparison',
         args: {
           profileIds: selectedProfileIds,
           startDate: getState().date.startDate,
           endDate: getState().date.endDate,
+          metric: 'audience',
```
Comment (msanroman) [inline packages/audience-comparison-chart/middleware.js]: As well in here, I think that having this set up as a constant could help giving it meaning and removing the impression that this is an arbitrary value.

File: packages/shared-components/ComparisonFooter/index.jsx [hunk 4]
```diff
@@ -37,10 +37,19 @@ const ProfileUsernameWrapper = styled.div`
   margin-right: 10px;
 `;
 
-const Footer = ({ profileTotals }) => (
+const ComparisonFooter = ({ profileTotals }) => (
   <Wrapper>
     <Grid>
-      {profileTotals.map(total => <GridItem
+      {profileTotals.map(total =>
+        // {
```
Comment (msanroman) [inline packages/shared-components/ComparisonFooter/index.jsx]: Woop, watch out for commented out code :) Maybe we can remove this?

File: packages/server/rpc/comparison/index.js [hunk 5]
```diff
@@ -4,24 +4,40 @@ const rp = require('request-promise');
...
 
-const METRICS_CONFIG = {
-  facebook: {
-    label: 'Fans',
-    color: '#ced7df',
-  },
-  instagram: {
-    label: 'Followers',
-    color: '#FEC78B',
+const METRIC_CONFIGS_BY_KEY = {
```
Comment (federicoweber) [inline packages/server/rpc/comparison/index.js]: how do you feel about switching the order of the indexes to `[profileService][metricKey]`? Super minor one but it's the same order we are using in other similar cases :smile:.

Comment (tiggreen) [reply packages/server/rpc/comparison/index.js]: I'm feeling great about that! Will address this in my next PR :) Feels good to have consistency all across analyze @federicoweber 😄 !

**Pull request merged**